src/operator/mod.rs

```diff
@@ -185,10 +185,12 @@ impl RadioOperator {
                         };
                     }
 
+                    let batch_size = 1000;
```

Comment:
how was this size determined? would like to understand if this is a magic number or if you had some analysis on the optimal size; either way would appreciate a comment in code

Reply:
magic number really :> i take recommendations but it seemed like an ok batch size